sysdeps/nacl/irt_syscalls.c

```diff
@@ -514,23 +513,9 @@ static int nacl_irt_getegid(void) {
 }
 
 static int nacl_irt_select_lind (int nfds, fd_set *readfds,
```

Comment:
I'm a bit confused by the `_lind` suffix here, since it doesn't appear in other function names? Is this right?

Reply:
This also appears in several calls, an example is socket call (line 521). I am going to check the other calls that we have implemented that still has the _lind suffix.

Reply:
You kept the lind stuff in the epoll calls in your PR. I think the purpose of it originally was for calls that weren't implemented by google in NaCl and that we were implementing special for lind, but we kind of stopped caring about that